types/date-arithmetic/tslint.json

```diff
@@ -1,11 +1,11 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
-        "ban-types": false,
-        "callable-types": false,
-        "dt-header": false,
-        "no-redundant-jsdoc": false,
-        "no-single-declare-module": false,
-        "prefer-method-signature": false
+        "no-duplicate-imports": [
```

Comment:
This need to be removed from here and fix the error

Reply:
I've tried that initially. If you mean the error `ERROR: 2:1  no-duplicate-imports  Multiple imports from 'date-arithmetic' can be combined into one.` which occurs when I remove this `no-duplicate-imports` config.
How should I combine the imports in a valid way?

As far as I know default imports with `import *` can't be combined with the destructuring syntax?
So this `import * as dateArithmetic, { Unit } from 'date-arithmetic';` isn't valid. What other ways do I have to fix this error?

I want both imports so I can test both commonjs and ecma script module usage since the target package supports both.

Reply:
Can you give me some direction @sheetalkamat